src/config/texts.js

```diff
@@ -14,3 +14,12 @@ export const texts = {
     policy: 'Datenschutz'
   }
 };
+
+import styled, { css } from 'styled-components/native';
+import { colors } from './colors';
+
+export const ScreenTitle = styled.Text`
```

Comment:
This has to be removed here and put somewhere in /components/. This is a styled `Text` component and not part of text configs.

Reply:
did it in new branch 'workin-on-components'